### ISSUE TITLE
rule MD013 ignore_code_blocks true doesn't always work correctly

### ISSUE THREAD

**Issue by konomikitten** [markdownlint/markdownlint#446]
## Description

I've managed to create a condition where MDL does not ignore long lines in code blocks.

### Environment

**MDL Version** 0.12.0

### Expected Behavior

MDL should ignore the long line.

### Actual Behavior
MDL suggests the line is too long even though it's in a code block.

## Replication Case

`~/.mdlrc`

```rb
style "#{File.expand_path('~/.mdl-default.rb')}"
```

`~/.mdl-default.rb`

```rb
rule 'MD013', :ignore_code_blocks => true
```

`~/.test.md`

````
```
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa-- []
```

a
```
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa-- []
```
a
````

Output from mdl

```
$ mdl ~/test.md
/home/debian/test.md:7: MD013 Line length
```

**Comment by Qup42** [comment]
I dug around in markdownlint for an unrelated issue. It turns out that the  second "code block" is parsed as a code span (inline code blocks) by kramdown. Judging by a comment in [markdownlint's Rule 031](https://github.com/updownpress/markdown-lint/blob/master/rules/031-blanks-around-fences.md?plain=1#L35-L36) this is the intended behaviour:
> Rationale: Aside from aesthetic reasons, some parsers, including kramdown, will not parse fenced code blocks that don't have blank lines before and after them.

**Comment by AJRepo** [comment]
Typo? Not reading that file? 

try mdl with the -v flag and see if (1) you get a parsing error or (2) you get the response "Loaded config from ..."

```
mdl -v ~/test.md
```
